src/v3/package.json

```diff
@@ -16,33 +16,16 @@
   "umd:main": "dist/index.umd.js",
   "module": "dist/index.m.js",
   "source": "components/widget/index.js",
-  "workspaces": {
-    "nohoist": [
-      "ajv-errors",
-      "@types/ajv-errors",
-      "@types/jest",
-      "preact",
-      "preact/**",
-      "preact-cli",
-      "preact-cli/**",
-      "preact-cli-plugin-env-vars",
-      "preact-cli-plugin-env-vars/**",
-      "preact-render-to-string",
-      "preact-render-to-string/**",
-      "preact-svg-loader",
-      "preact-svg-loader/**"
-    ]
-  },
   "scripts": {
-    "build": "preact build --no-prerender --no-sw --dest ../../dist/dist/js",
+    "build": "vite build",
     "clean-svg": "svgo -f src/img/authCoinIcons",
-    "codegen": "yarn workspace @okta/okta-signin-widget codegen",
-    "dev": "yarn codegen && preact watch --no-prerender --no-sw --port 3000",
+    "codegen": "echo 'codegen'",
```

Comment:
Are we just stubbing this out in case we still need it at some point?

Reply:
I'll fix/revert it. Just needed to skip the codegen problem while troubleshooting TestCafe and can't leave the comment in `package.json`

Reply:
reverted to use codegen from project root